client/my-sites/sharing/connections/service-examples.jsx

```diff
@@ -205,6 +206,25 @@ class SharingServiceExamples extends Component {
 		];
 	}
 
+	google_my_business() {
+		return [
+			{
+				image: {
+					src: '/calypso/images/google-my-business/phone-screenshot-cropped.png',
+					alt: this.props.translate( 'Manage Google My Business locations', { textOnly: true } ),
```

Comment:
I'm not sure, I would have thought it made sure the object returned by translate is a string and not a component (basically bypassing `interpolate-components`) but I couldn't find this option used anywhere either. Guess I'll remove it here

Reply:
Oh wait those are `alt` properties, better be safe than sorry and avoid injections.
@akirk any idea if this `textOnly` option is used in `i18n-calypso` or somewhere else?

Reply:
It's no longer used. Its purpose is that the Community Translator might cause this to return a React Component instead of a string. Therefore if the alt property is checked using a `PropType.String` it should be replaced with `TranslatableString` instead. See #21917